integration/benchmark/cmd/ci/main.go

```diff
@@ -71,7 +71,10 @@ func main() {
 	// CI relevant flags
 	tpsFlag := flag.String("tps", "300", "transactions per second (TPS) to send, accepts a comma separated list of values if used in conjunction with `tps-durations`")
 	tpsDurationsFlag := flag.String("tps-durations", "10m", "duration that each load test will run, accepts a comma separted list that will be applied to multiple values of the `tps` flag (defaults to infinite if not provided, meaning only the first tps case will be tested; additional values will be ignored)")
-	ciFlag := flag.Bool("ci-run", false, "whether or not the run is part of CI")
+	_ = flag.Bool("ci-run", false, "(deprecated)")
```

Comment:
This is only referenced in integration/localhost/Makefile, we should remove it both in that file and here entirely.